src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -79,23 +79,10 @@ public override void ExecuteCmdlet()
                                     "Consider using \"latest\" as the image version. This allows VMSS to auto upgrade when a newer version is available.");
                             }
 
-                            if (parameters?.OrchestrationMode == "Flexible")
-                            {
-                                if (parameters?.VirtualMachineProfile?.NetworkProfile?.NetworkInterfaceConfigurations != null)
-                                {
-                                    foreach (var nicConfig in parameters.VirtualMachineProfile.NetworkProfile.NetworkInterfaceConfigurations)
-                                    {
-                                        if (nicConfig.IpConfigurations != null)
-                                        {
-                                            foreach (var ipConfig in nicConfig.IpConfigurations)
-                                            {
-                                                ipConfig.LoadBalancerInboundNatPools = null;
-                                            }
-                                        }
-                                    }
-                                }
+                            if (parameters.OrchestrationMode == null) { parameters.OrchestrationMode = flexibleOrchestrationMode; }
```

Comment:
small detail but does parameters need to be checked for null (Parameters?)

Reply:
not necessary it is always being set in line 74

Reply:
please approve !